src/components/picker/index.tsx

```diff
@@ -196,7 +191,7 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
     closeExpandable,
     toggleExpandable
   }) => {
-    if (renderCustomModal) {
+    if (componentProps.customModal) {
```

Comment:
Don't you need to check if type === 'custom' as well?

Reply:
I'm not sure, the main idea was when the picker is set to type `custom` only then the user can pass `customModal`.
I'll add the type check for now.

Reply:
Yes. That's what I mean... If the user is not passing type='custom' don't run this. If you don't check this type it will render if there's no type (once you take out the fallback in the hook). Now it works "properly" with other types, like "modal", since the modal case in the switch is placed before the custom, otherwise you'll run this even if the type passed is not "custom"